nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const router = express.Router();
+const meals = require("../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
+    console.log(meals);
+    console.log("in /api/meals");
+    // to verify for empty object
+  if (
+    Object.keys(request.query).length === 0 &&
```

Comment:
Before using request query parameters, you need to define them.
Unless you define them as object, you cannot use object.keys!

Reply:
for ex:  `const { maxPrice, limit } = request.params;`